src/Records/MX.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Spatie\Dns\Records;
+
+/**
+ * @method int pri()
+ * @method string target()
+ */
+class MX extends Record
+{
+    protected int $pri;
+    protected string $target;
+
+    public static function parse(string $line): self
+    {
+        $attributes = static::lineToArray($line, 6);
+
+        return static::make([
+            'host' => $attributes[0],
+            'ttl' => $attributes[1],
+            'class' => $attributes[2],
+            'type' => $attributes[3],
+            'pri' => $attributes[4],
+            'target' => $attributes[5],
+        ]);
+    }
+
+    public function __toString(): string
+    {
+        return sprintf(
+            "%s.\t\t%d\t%s\t%s\t%d\t%s.",
+            $this->host,
+            $this->ttl,
+            $this->class,
+            $this->type,
+            $this->pri,
+            $this->target
+        );
+    }
+
+    protected function castPri($value): string
```

Comment:
It's not clear to me what `pri` means? Primitive? Let's not use abbreviations in method names.

Reply:
I've used the exact names like in DNS and plain PHP (also for mapping) in this case `pri` is the MX record priority.
If I change this I will have to add a naming mapping and can't use the current auto/magic logic.

Reply:
Ok, let's keep it like it is